app/uk/gov/hmrc/agentsubscriptionfrontend/views/subscribe.scala.html

```diff
@@ -15,12 +14,23 @@
   @form(
     action = routes.SubscriptionController.submitKnownFacts,
     'class -> "form js-form") {
-    <fieldset class="form-field-group">
-      @helper.inputText(knownFactsForm("utr"))
-      @helper.inputText(knownFactsForm("postcode"))
+
```

Comment:
I lifted the UI code from agent-client-authorisation-frontend and it uses (HMRC) play-ui helpers instead of using Play's helpers directly. Presumably in order to generate HTML that matches the CSS but I'm not sure.